asf-site-src/pom.xml

```diff
@@ -45,74 +45,6 @@
         <artifactId>yetus-maven-plugin</artifactId>
         <version>${project.version}</version>
         <executions>
-          <!-- AUTOMATED_EDIT_BEGIN -->
-          <execution>
-            <id>0.11.1</id>
```

Comment:
These are no longer required because we now preserve the existing previously-built site files and only overlay on the new version files?

Reply:
Basically yes.

The previous order was basically:

1. Generate inprogress javadocs
2. Generate inprogress apidocs
3. Extract older version archives <- where everything fell apart
4. Create new releasedocs for all versions
5. Run middleman 

By moving step 3 to be after step 5, changing the directory to where the extract should actually live, changing where the inprogress javadoc should live, and preventing overwriting the "existing" files (i.e., the changelog info), a big chunk of this hocus pocus goes away entirely.